src/content.js

```diff
@@ -10,7 +12,13 @@ export const mountContentScript = (contentScript) => {
   window.addEventListener('load', async () => {
```

Comment:
I would add it to the consts
```suggestion
  window.addEventListener(EVENT_ON_LOAD, async () => {
```

Reply:
Yes, the naming is the thing that bothers me the most

Reply:
Ok. Will add more naming suggestions shortly

Reply:
But specifically, in this case, I prefer to keep it hard-coded.

This `"load"` event is not our internal contract, but it is a real `window` event, so its value itself is important and there is no reason to go to another file just to see the real value (in contrast to the other events, which their value is not important, and the only important thing is we are using the same variable in both dispatch and listener).